modules/kafka/src/it/scala/com/snowplowanalytics/snowplow/enrich/kafka/test/KafkaSink.scala

```diff
@@ -0,0 +1,37 @@
+package com.snowplowanalytics.snowplow.enrich.kafka.test
+
+import cats.Parallel
+import cats.implicits._
+import cats.effect.{ConcurrentEffect, ContextShift, Resource, Timer}
+import com.snowplowanalytics.snowplow.enrich.common.fs2.config.io.Output
+import fs2.kafka.{KafkaProducer, ProducerSettings}
+import fs2.{Pipe, Stream}
+
+import java.util.UUID
+
+object KafkaSink {
+
+  def init[F[_]: ConcurrentEffect: ContextShift: Parallel: Timer](topicName: String, bootstrapServers: String): Pipe[F, Array[Byte], Unit] = {
```

Comment:
Instead of redefining a Kafka sink it would be great to re use [the one that you already have](https://github.com/snowplow/enrich/blob/enrich-kafka/modules/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.kafka/Sink.scala#L27).

I did that in [enrich kinesis integration test](https://github.com/snowplow/enrich/blob/enrich-kafka/modules/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.kafka/Sink.scala#L27)

Reply:
> could you explain more please?

What you need for the integration tests is something that can insert an `Array[Byte]` into Kafka. This is exactly what [this](https://github.com/snowplow/enrich/blob/enrich-kafka/modules/kafka/src/main/scala/com.snowplowanalytics.snowplow.enrich.kafka/Sink.scala#L30) gives you.

It's great to reuse it for 2 reasons :

- Re use existing pieces instead of reinventing the wheel
- The more we use this code the more chances we have to find bugs

> if we have a bug in main impl, aren't we going to get tricked by successful integration test?

If we have a bug in main impl, integration tests won't be successful